src/node_crypto.cc

```diff
@@ -5227,6 +5235,15 @@ void PublicKeyCipher::Cipher(const FunctionCallbackInfo<Value>& args) {
   uint32_t padding;
   if (!args[offset + 1]->Uint32Value(env->context()).To(&padding)) return;
 
+  const EVP_MD* oaep_md = nullptr;
+  if (padding == RSA_PKCS1_OAEP_PADDING && !args[offset + 2]->IsUndefined()) {
+    const node::Utf8Value oaep_hash(env->isolate(), args[offset + 2]);
+    oaep_md = EVP_get_digestbyname(*oaep_hash);
+    if (oaep_md == nullptr) {
+      return env->ThrowError("Unknown OAEP message digest");
```

Comment:
It would be better if this had a .code. I wonder if emulating the error structure of `EVP_PKEY_CTX_md` would be possible, ie, setting the standard .code, ... values to something similar to what `EVPerr(EVP_F_EVP_PKEY_CTX_MD, EVP_R_INVALID_DIGEST)` would do if it was being used here? Also,  would it actually be possible to use `EVP_PKEY_CTX_md()` here? I found it by attempting encryption with the "sam" digest with CLI: `openssl pkeyutl -inkey rsa.priv -encrypt -pkeyopt rsa_oaep_md:sam -in /etc/passwd`

Reply:
any comment?

Reply:
Sorry Sam, I must have missed these comments. Addressed in 96a4bc6a7cfa9a8379d855c2a0fedb61b8b6d977.